.github/workflows/testing.yml

```
@@ -263,12 +287,21 @@ jobs:
         id: get-date
         run: |
           echo "DATE=$(powershell get-date -format "yyyyMMdd")" | Out-File -FilePath $env:GITHUB_OUTPUT -Encoding utf8 -Append
-      - name: Get cached database
+          echo "YESTERDAY=$(get-date (get-date).addDays(-1)  -format "yyyyMMdd")" | Out-File -FilePath $env:GITHUB_OUTPUT -Encoding utf8 -Append
```

```suggestion
          echo "YESTERDAY=$(powershell get-date (get-date).addDays(-1)  -format "yyyyMMdd")" | Out-File -FilePath $env:GITHUB_OUTPUT -Encoding utf8 -Append
```

I don't know about github actions, but it won't work on my local machine without `powershell` being explicitly called.